src/main/java/org/craftercms/studio/api/v2/service/config/ConfigurationService.java

```diff
@@ -30,4 +35,10 @@
 
     Map<String, List<String>> geRoleMappings(String siteId) throws ConfigurationException;
 
+    String loadConfiguration(String siteId, String module, String location, String environment);
+
+    Document loadConfigurationDocument(String siteId, String module,  String location, String environment)
```

Comment:
Who uses this method? Why do they want the configuration as a DOM? (Let's discuss so we can plan the future where it's all YAML.)

Reply:
API v1 services are using DOM to extract properties. I wanted to have SAX parsing in a single class. Future refactoring and possible switching to yaml will be easier this way.